templates/components/common/quick-search.html

```diff
@@ -14,7 +14,7 @@
             >
         </div>
     </form>
-    <section class="js-quick-search-results" data-bind="html: results"></section>
+    <section class="js-quick-search-results c-quick-search-results" data-bind="html: results"></section>
```

Comment:
@hectorfhurtado same here. Please append the `c-quick-search-results` as a first class.